lib/src/components/NxTag/NxTag.scss

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold();
+  align-items: center;
+  background-color: $nx-blue-800;
+  border: 1px solid $nx-blue-800;
+  border-radius: 16px;
+  box-sizing: border-box;
+  color: $nx-text-color-dark;
+  cursor: default;
+  display: inline-block;
+  font-size: $nx-font-size-s;
+  margin: 0 $nx-spacing-xs $nx-spacing-xs 0;
+  padding: 5px $nx-spacing-xs 6px $nx-spacing-md;
+  max-width: 824px;
+}
+
+.nx-tag__text {
+  @include nx-truncate-ellipsis;
+  padding-right: $nx-spacing-xs;
+}
+
+.nx-tag--selectable {
+  cursor: pointer;
+  display: inline-flex;
+
+  &:focus {
```

Comment:
Strictly speaking, zeplin does not show focus styles for the tags, only for the tag menu (which we aren't doing yet).  Did you confirm with @dturner32 how this is supposed to look?

Reply:
> Strictly speaking, zeplin does not show focus styles for the tags, only for the tag menu (which we aren't doing yet). Did you confirm with @dturner32 how this is supposed to look?

Once we decided that all tags (regardless of colour) could be selectable I assumed that these states would apply to all tags regardless of colour: 
<img width="286" alt="Screen Shot 2021-01-22 at 12 51 48 PM" src="https://user-images.githubusercontent.com/1991443/105526673-b7a55f80-5cb0-11eb-91a4-4ace6f0db564.png">

Reply:
correct, menu tags would reflect the same states that all the other tags get